def raise_exception():
    raise NameError(message="This is a type exception.")
print("This is a type exception")